pkg/streamer/hub_test.go

```diff
@@ -85,3 +120,48 @@ func (s *testStreamerSuite) TestRelayLogInfoHub(c *C) {
 	c.Assert(taskName, Equals, "")
 	c.Assert(earliest, IsNil)
 }
+
+func (t *testHubSuite) TestReaderHub(c *C) {
+	h := GetReaderHub()
```

Comment:
how about GetReaderHub again, and check these two is one

Reply:
done in https://github.com/pingcap/dm/pull/140/commits/dc204eef6715bf3e0187b1e2ce410a7cee4b5549